examples/typescript/external_signing.ts

```diff
@@ -47,7 +48,7 @@ const balance = async (aptos: Aptos, account: Account, name: string): Promise<nu
  * Provides a mock "Cold wallet" that's signed externally from the SDK
  */
 class ExternalSigner {
-  private account: Account;
+  private account: Ed25519Account;
```

Comment:
why does this need to change to Ed25519Account?

Reply:
Looking at the code below (not in the changeset) `AccountAuthenticatorEd25519` is used specifically.
There's also a hard cast for the public key to `Ed25519PublicKey`.
I thought it was unnecessary to keep the property as the generic `Account`.

Reply:
I guess in my mind, an ExternalSigner interface need not commit to any particular signing scheme.

I think similar to Account, we should provide a generic constructor for AccountAuthenticator and update the example to be generic to the scheme.